ods_ci/tests/Tests/1300__model_registry/1302_model_registry_model_serving.robot

```diff
@@ -0,0 +1,277 @@
+    Open With JupyterLab Menu  Run  Restart Kernel and Run All Cells…
+    Click Element    xpath=//div[contains(text(),"Restart") and @class="jp-Dialog-buttonLabel"]                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                
+    Wait Until JupyterLab Code Cell Is Not Active  timeout=120s
+    Sleep    2m    msg=Waits until the jupyter notebook has completed execution of all cells
+    JupyterLab Code Cell Error Output Should Not Be Visible
+    SeleniumLibrary.Capture Page Screenshot
+
+Wait For Model Registry Containers To Be Ready
+    [Documentation]    Wait for model-registry-deployment to be ready
+    ${result}=    Run Process
+    ...        oc wait --for\=condition\=Available --timeout\=5m -n ${PRJ_TITLE} deployment/model-registry-db
+    ...        shell=true    stderr=STDOUT
+    Log To Console    ${result.stdout}
+    ${result}=    Run Process
+    ...        oc wait --for\=condition\=Available --timeout\=5m -n ${PRJ_TITLE} deployment/model-registry-deployment
+    ...        shell=true    stderr=STDOUT
+    Log To Console    ${result.stdout}
+
+Get Domain
+    [Documentation]  Gets the Domain and returns it to 'Run Oc Commands And Capture Output'.
+    # Run the command to get the ingress domain
+    ${domain_result}=    Run Process    oc    get    ingresses.config/cluster
+    ...    -o    yaml    stdout=PIPE    stderr=PIPE
+    ${rc}=    Set Variable    ${domain_result.rc}
+    IF    $rc > 0    Fail    Command 'oc whoami -t' returned non-zero exit code: ${rc}
+    ${domain_yaml_output}=    Set Variable    ${domain_result.stdout}
+
+    # Return the domain from stdout
+    ${domain_parsed_yaml}=    Evaluate    yaml.load('''${domain_yaml_output}''', Loader=yaml.FullLoader)
+    ${ingress_domain}=    Set Variable    ${domain_parsed_yaml['spec']['domain']}
+
+    # Return both results
+    RETURN    ${ingress_domain}
+
+Get Token
+    [Documentation]    Gets the Token and returns it to 'Run Oc Commands And Capture Output'.
+    ${token_result}=    Run Process    oc    whoami    -t    stdout=YES
+    ${rc}=    Set Variable    ${token_result.rc}
+    IF    ${rc} > 0    Fail    Command 'oc whoami -t' returned non-zero exit code: ${rc}
+    ${token}=    Set Variable    ${token_result.stdout}
+    RETURN    ${token}
+
+Create OpenShift Project
+    [Documentation]    Creates a test namespace and tracks it via ServiceMesh
```

Comment:
"tracks it via Service Mesh"
How does it do that? this seems to only be creating a new project if it doesn't exist already. Did you want to apply the SMM file as well?

Reply:
Yes, the SMM file applied earlier in the set up

Reply:
Yeah, given the documentation my question was if you actually wanted to include the call to `Apply ServiceMeshMember Configuration` inside this keyword, rather than calling them separately from the test case.

Reply:
Ahh yes, that makes sense